### ISSUE TITLE
a11y: Color Contrast - Repos

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13738]
Insufficient color contrast for the borders of form controls 

## Actual Result

Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

### Repository: Create

"Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1
Main Content
- e.g. foo and e.g. bar input fields (which gets available after activating ""Add Label"" and ""Add Annotation"" buttons)"

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
- Remove button (available after activating Add Label and Add Annotation button)
- Cancel Button (on default state )
- Name, Description, Index URL and so on input fields. (on hover and on focus)

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1 
Main Content
- Placeholder text such as A unique name, Any text you want that better describes this resource, e.g. https://charts.rancher.io, e.g. foo and e.g. bar

Foreground Text Color: #3D98D3
Background Text Color: #E2F0F8
Ratio: 2.72:1
Main Content
- Add Label and Add Annotation buttons

Foreground Text Color: #FFFFFF 
Background Text Color: #297DB4 
Ratio: 4.48:1
Main Content
- Cancel, Create,  Add Label and  Add Annotation button (on focus) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Create a SSH Key Secret, Create a HTTP Basic Auth Secret and None (dropdown option in selected state when keyboard focus is on option and on hover in ""Authentication"" drop-downs)

Foreground color: #FFFFFF
Background color: #B6B6C2
Contrast ratio: 2.00:1 
Main Content: 
- Create a SSH Key Secret  (dropdown option in selected state when keyboard focus is not on option in ""Authentication"" drop-down)"

### Git Repo - Add Repository

"Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1
Main Content 
- Path input field (available after activating Add Path button)  "

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1 
Main Content
- Git Repo:
- Repository Details
- Cancel and Edit as YAML (on default state) 
- Name, Description, Branch Name, Repository URL input field and so on input fields (on hover and focus)
-Git Authentication, Helm Authentication, TLS Certificate Verification dropdown (on hover and focus)

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- A Branch, A Revision, None, Create a HTTP Basic Auth Secret, Create a SSH Key Secret and so on (dropdown option in selected state when keyboard focus is on option and on hover in ""Watch, Git Authentication, Helm Authentication, TLS Certificate Verification"" drop-downs)

Foreground color: #FFFFFF
Background color: #B6B6C2
Contrast ratio: 2.00:1 
Main Content: 
- Create a SSH Key Secret, None, Require a valid certificate,  (dropdown option in selected state when keyboard focus is not on option in ""Authentication"" drop-down)

Foreground Text Color: #3D98D3
Background Text Color: #E2F0F8
Ratio: 2.72:1
Main Content
- Add Path       

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1 
Main Content
- A unique name
- Any text you want that better describes this resource
- e.g. https://github.com/rancher/fleet-examples.git or git@github.com:rancher/fleet-examples.git
- e.g. /directory/in/your/repo
           

                  
          "

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 189 Fails VPAT. No Notes. Original Comment:

> Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 190 fails VPAT. 

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF 
> Ratio: 1.78:1 
> Main Content
> - Placeholder text such as A unique name, Any text you want that better describes this resource, e.g. https://charts.rancher.io, e.g. foo and e.g. bar
> 
> Foreground Text Color: #FFFFFF
> Background Text Color: #B6B6C2
> Ratio: 2.0:1
> Main Content
> -  None (dropdown option in selected state when keyboard focus is not on option in ""Authentication"" drop-downs) 
> 
> Foreground Text Color: #3B71E1
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> - Cancel button (on focus and hover)
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 196 fails VPAT. No notes. Original comment:

> Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 197 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground color: #FFFFFF
> Background color: #B6B6C2
> Contrast ratio: 2.00:1 
> Main Content: 
> - Create a SSH Key Secret, None, Require a valid certificate,  (dropdown option in selected state when keyboard focus is not on option in ""Authentication"" drop-down)
> 
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF 
> Ratio: 1.78:1 
> Main Content
> - A unique name
> - Any text you want that better describes this resource
> - e.g. https://github.com/rancher/fleet-examples.git or git@github.com:rancher/fleet-examples.git
> - e.g. /directory/in/your/repo
> 
> Foreground color: #2F68DF
> Background color: #EDF2FC
> Contrast ratio: 4.4:1
> (On hover on focus )
> - Cancel
> - Edit as YAML  
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."
